client/src/pages/RegisterPage.jsx

```diff
@@ -6,6 +6,24 @@ import axios from "axios";
 import { AuthContext } from "../context/AuthContext";
 import CustomInput from "../components/CustomInput";
 
+import {Avatar, Button, CssBaseline, Link, Grid, Box, Typography, Container } from "@mui/material";
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+function Copyright(props) {
+  return (
+      <Typography variant="body2" color="text.secondary" align="center" {...props}>
+          {'Copyright © '}
+          <Link color="inherit" href="http://localhost:3000/">
+              KBU_Sports
+          </Link>{' '}
```

Comment:
```suggestion
          <Link color="inherit" to="/">
              KBU_Study
          </Link>{' '}
```
1. `to` 옵션이 적용되지 않은 부분은 LoginPage에도 있습니다.
2. 텍스트가 KBU_Study가 아닌곳이 존재한다면 변경해주세요.

Reply:
to="/"를 사용하였더니 링크가 먹지 않는 이슈가 있어 href="/"를 사용하였습니다. 조금 더 해보고 다시 PR드리겠습니다.